module-2/backend/week_two.md

```diff
@@ -16,8 +18,11 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+* .all, .first, .last, .(any other column name in the associated table). class Team inherits from Base in the active record module, which provides Team with the base active record methods.
 
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+* Team.find(4).name
+* Owner.find(Team.find(4))
```

Comment:
you would need an integer to call `Owner.find(something)`, and `Team.find(4)` will return a Team object. What else would you need here?